vllm/model_executor/parallel_utils/pynccl.py

```diff
@@ -219,25 +220,22 @@ def __init__(
                                     store=store,
                                     group_name=group_name,
                                     pg_options=pg_options)
-        self.world_size = dist.get_world_size()
-        self.rank = dist.get_rank()
```

Comment:
We cound still save `self.world_size/self.rank/self.local_rank` , for convenience.